docs/guides/dws-on-csm/readme.md

```diff
@@ -0,0 +1,185 @@
+```
+
+### Restart the Slurm Configuration Job
+
+The Slurm configuration job will process the ConfigMap into a second one that is included in the pod spec.  The following steps to reconfigure Slurm are found in the HPE Cray Programming Installation Guide.
+
+```bash
+ncn-m001:~ # JOBNAME=$(kubectl get job -n services | grep slurm-config | grep -v import | awk '{print $1}')
+ncn-m001:~ # kubectl get job -n services -o yaml $JOBNAME > slurm-config.yaml
+ncn-m001:~ # kubectl delete -f slurm-config.yaml
+ncn-m001:~ # yq d -i slurm-config.yaml spec.template.metadata
+ncn-m001:~ # yq d -i slurm-config.yaml spec.selector
+ncn-m001:~ # kubectl apply -f slurm-config.yaml
+ncn-m001:~ # SLURMCTLD_POD=$(kubectl get pod -n user -lapp=slurmctld -o jsonpath='{.items[0].metadata.name}')
+ncn-m001:~ # kubectl exec -n user $SLURMCTLD_POD -c slurmctld -- scontrol reconfigure
+```
+
+After the configuration job has been restarted, wait about 30 seconds to see the new content appear inside the pod.
+
+If you were only adding or modifying burst_buffer.lua, then Slurm is now ready for your jobs.  If you were also modifying slurm.conf to set the BurstBufferType, or you were adding or modifying burst_buffer.conf, then you must also restart the Slurm pod.
+
+```bash
+ncn-m001:~ # kubectl rollout restart deployment -n user slurmctld
+```
+
+## Submit a test job
+
+Use a UAN host to submit a test job.  The following assumes a UAN host named `uan01` and a user account named `vers`.
+
+```bash
+ncn-m001:~ # ssh uan01
+uan01:~ # su vers
+vers@uan01:/root> cd /lus/vers
+vers@uan01:/lus/vers> cat > /tmp/dws-test <<EOF
+#!/bin/sh                
+#SBATCH --time=1
+#DW
+/bin/hostname
+srun -l /bin/hostname
+srun -l /bin/pwd
+EOF
+```
+
+Note that we will submit the test job from a directory where our `vers` account can write its output files.
+
+```bash
+vers@uan01:/lus/vers> sbatch /tmp/dws-test
+Submitted batch job 7773
+vers@uan01:/lus/vers> sacct | grep 7773
+vers@uan01:/lus/vers> cat slurm-7773.out
+```
+
+Get the status of the workflow.
+
+```bash
+vers@uan01:/lus/vers> scontrol show bbstat workflow 7773 && echo
+desiredState=DataIn currentState=DataIn status=Completed
+```
+
+Note that the workflow resource will no longer exist after Slurm has completed its teardown state for this job.
+
+## Inspect DWS Workflow resources
+
+Inspect DWS Workflow resources by using the fully-qualified CRD name.
+
+```bash
+ncn-m001:~ # kubectl get workflows.dws.cray.hpe.com -A
+```
+
+<details>
+<summary>Multiple CRDs named "Workflow"</summary>
+    The fully-qualified name will protect you from picking up any resource named "Workflow" that belongs to some other service.
+
+Your system may have multiple CRDs named "Workflow".
+
+```bash
+ncn-m001:~ # kubectl get crds | grep -E '^workflows\.'
+workflows.argoproj.io                            2022-11-25T02:56:48Z
+workflows.dws.cray.hpe.com                       2022-11-21T15:38:13Z
+```
+</details>
+
```

Comment:
I'd also add a Troubleshooting section. In particular, I'd add a way to grep for the lua messages from the slurmctld pod, and also a way to get the dws logs. These are the things I'd expect them to grab for us if something goes wrong and we need to get more information.